src/Pokemon.java

```diff
@@ -75,7 +75,7 @@ public int getSchade() {
     }
 }
 class WaterPokemon extends Pokemon{
```

Comment:
Misschien was het overzichtelijker als je alles een eigen file had gegeven, Nu is het moeilijk leesbaar en krijg je dus het probleem met dat je sommige dingen package private moet maken of juist public.

Reply:
Ja had gekund. Dat was een beetje luiheid van mijn kant.